### ISSUE TITLE
Error when viewing settings page

### ISSUE THREAD

**Issue by chrisburton** [jrc9designstudio/google-analytics-v2-hub#4]
I received an error upon installing and visiting the settings page. Specifically `width:50` in `settings.yaml`. The cause for this error is a missing space. 

**source:** `/site/addons/GoogleAnalytics/settings.yaml`

**Error:**

```yaml
link_id:
    type: toggle
    width:50
```

**Fix:**
```yaml
link_id:
    type: toggle
    width: 50
```

**Comment by jcohlmeyer** [comment]
@chrisburton can you share the error and any relevant logs with me?

**Comment by jcohlmeyer** [comment]
Fixed in [2.2.1](https://github.com/jcohlmeyer/statamic-google-analytics/releases/tag/2.2.1)
